openreview/venue/venue.py

```diff
@@ -69,7 +69,6 @@ def __init__(self, client, venue_id, support_user):
         self.senior_area_chair_identity_readers = []
         self.automatic_reviewer_assignment = False
         self.decision_heading_map = {}
-        self.use_publication_chairs = False
```

Comment:
shouldn't we keep this property and ask in the request form if the venue has publication chairs. We don't want to create the publication chairs group for venues that don't have it

Reply:
I was thinking to add a question with "yes" or "no" asking if the venue has publication chairs but don't ask the emails because they might not now the emails at the moment of submitting the request form.